setup.py

```diff
@@ -28,12 +24,10 @@
         'Topic :: Scientific/Engineering :: GIS',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.5',
-        'Programming Language :: Python :: 2.6',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.4'
+        'Programming Language :: Python :: 3.5'
+        'Programming Language :: Python :: 3.6'
         ],
-      license='3-Clause BSD',
+      license='2-Clause BSD',
```

Comment:
This might be less of a concern, but is it better to follow the 3-Clause BSD license suggested in the [submodule_template](https://github.com/pysal/submodule_template/blob/master/LICENSE)?

Reply:
Good point, that I didn't consider. Should I go ahead and make the change or wait for Taylor's input since he is the maintainer?

Reply:
Sure. Let's wait for Taylor's input!